app/classifier/index.cjsx

```diff
@@ -17,12 +17,17 @@ workflowAllowsSeparateFrames = require '../lib/workflow-allows-separate-frames'
 WorldWideTelescope = require './world_wide_telescope'
 MiniCourseButton = require './mini-course-button'
 GridTool = require './drawing-tools/grid'
+Intervention = require '../lib/intervention'
+experimentsClient = new (require '../lib/experiments-client')
+
+PULSAR_HUNTERS_SLUG = 'zooniverse/pulsar-hunters'
```

Comment:
Might need to do a rebase with master. This line should be removed with this merged PR: https://github.com/zooniverse/Panoptes-Front-End/pull/2923